workers/ohsome_quality_analyst/api/api.py

```diff
@@ -108,13 +110,17 @@ async def validation_exception_handler(
 
 @app.exception_handler(OhsomeApiError)
 @app.exception_handler(SizeRestrictionError)
+@app.exception_handler(RasterDatasetNotFoundError)
+@app.exception_handler(RasterDatasetUndefinedError)
```

Comment:
Why do you deal with RasterDatasets in this PR? I think this is added by mistake in this PR.

Reply:
Thats right. Removed those lines.